src/Engine/Graphics/RenderBase.cpp

```diff
@@ -480,8 +471,8 @@ void RenderBase::TransformBillboard(SoftwareBillboard *pSoftBillboard, RenderBil
     billboard->pQuads[1].texcoord.x = 0.f;
     billboard->pQuads[1].texcoord.y = 1.f;
 
-    v14 = (float)((int)pSprite->uAreaWidth + pSprite->uAreaX + pSprite->uBufferWidth / 2 - pSprite->uBufferWidth);
-    v15 = (float)((int)pSprite->uBufferHeight - pSprite->uAreaHeight - pSprite->uAreaY);
+    v14 = (float)((int)pSprite->uWidth + pSprite->uAreaX + pSprite->uWidth / 2 - pSprite->uWidth);
+    v15 = (float)((int)pSprite->uHeight - pSprite->uHeight - pSprite->uAreaY);
```

Comment:
uHeight - uHeight?
Are they supposed to be same?

Reply:
Hmm.....